components/tree-grid/__examples__/nested.jsx

```diff
@@ -0,0 +1,156 @@
+import React from 'react';
+import TreeGrid from '~/components/tree-grid';
+import IconSettings from '~/components/icon-settings';
+
+const data = {
+	cols: [
+		{
+			id: '1',
+			label: 'Account Name',
+			href: 'javascript:void(0);',
+		},
+		{
+			id: '2',
+			label: 'Employees',
+			href: 'javascript:void(0);',
+		},
+		{
+			id: '3',
+			label: 'Phone Number',
+			href: 'javascript:void(0);',
+		},
+		{
+			id: '4',
+			label: 'Account Owner',
+			href: 'javascript:void(0);',
+		},
+		{
+			id: '5',
+			label: 'Billing City',
+			href: 'javascript:void(0);',
+		},
+	],
+	rows: [
+		{
+			id: '1',
+			cols: [
+				{
+					label: 'Rewis Inc',
+				},
+				{
+					label: '3100',
+				},
+				{
+					label: '837-555-1212',
+				},
+				{
+					label: 'Jane Doe',
+					href: 'javascript:void(0);',
+				},
+				{
+					label: 'Phoenix, AZ',
+				},
+			],
+		},
+		{
+			id: '2',
+			childOpen: true,
+			cols: [
+				{
+					label: 'Acme Corporation',
+				},
+				{
+					label: '10000',
+				},
+				{
+					label: '837-555-1212',
+				},
+				{
+					label: 'Jane Doe',
+					href: 'javascript:void(0);',
```

Comment:
The default cell render should be just a string. If you'd like to scope this project to only output that I'm cool with that.

`DataTable` has a `CustomCell` render that receives the tree data as props. The `CustomCell` is the child of the Column child. See https://github.com/salesforce/design-system-react/blob/master/components/data-table/__examples__/advanced.jsx#L10